workflows/push-images.yml

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2025 Canonical Ltd.
+name: Push Images
+
+on:
+  workflow_call:
+    inputs:
+      branch-name:
+        description: Name of the branch to checkout
+        required: false
+        type: string
+        default: ${{ github.ref_name }}
+
+jobs:
+  push-images:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'omec-project'
+    env:
+      REGISTRY: registry.aetherproject.org
```

Comment:
If these are meant to be reusable, I think all of these env vars should be inputs. These would be fine as default values, but it will be significantly more flexible if they can be set by the calling workflow.